.github/workflows/publish.yml

```diff
@@ -73,20 +76,29 @@ jobs:
         - cp39*macosx_x86_64
         - cp310*macosx_x86_64
         - cp311*macosx_x86_64
+        - cp312*macosx_x86_64
 
         - cp39*macosx_arm64
         - cp310*macosx_arm64
         - cp311*macosx_arm64
+        - cp312*macosx_arm64
 
         # Windows wheels
 
         - cp39*win32
         - cp310*win32
         - cp311*win32
+        - cp312*win32
 
         - cp39*win_amd64
         - cp310*win_amd64
         - cp311*win_amd64
+        - cp312*win_amd64
+
+      # Required so that cp312 can grab a pre-release numpy.
+      # Can remove when cp312 is released.
+      env: |
+        CIBW_ENVIRONMENT: PIP_PRE=1
```

Comment:
This would break all the win32 wheels because it pulls down numpy 1.26b for everything. Not sure if we really want it like this.

Reply:
I guess some manylinux_x86_64 too 🤷